src/aws/terraform_shared_resources/state.tf

```diff
@@ -0,0 +1,12 @@
+provider "aws" {
```

Comment:
the folder should not contain `terraform` imo. The other folder is called `create_state_bucket` and everything is terraform here... just `shared_resources` seems enough?

Reply:
Ok I also renamed the binary script in `shared-resources`